app/routes_goals.py

```diff
@@ -0,0 +1,93 @@
+from app import db
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request
+from app.routes_tasks import Task
+
+from app.route_helper import get_one_obj_or_abort
+
+from app.slack_helper import sendSlackNotification
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Good use of try-except, but it would be better to let the user know what data specifically was invalid. 

This could also be made a helper function and shared with the `patch` request.